### ISSUE TITLE
Adapt Accessibility tests: co-05 hot graphic component buttons lack unique descriptive text

### ISSUE THREAD

**Issue by shauncg** [adaptlearning/adapt_framework#408]
The hotspots in the 'Key Facts' component on co-05 are both read as "alt text button".

A more specific description would be helpful for the user. E.g. "Hotspot 1", "Hotspot 2".
